docs/api/field_type_reference.md

```diff
@@ -412,6 +412,29 @@ $countryValue = new Country\Value(
 );
 ```
 
+!!! dxp
+
+    ## DAM Field Type
+
+    This Field Type stores asset information from a DAM system.
```

Comment:
DAM - to pewnie jasne dla każdego, ale nie znalazłem w docsach rozwinięcia tego akronimu. Myślę, że przydałoby się przy pierwszym wystąpieniu, ew. również w release notesach rozwinąć go do pełnej nazwy:
"Data Asset Management (DAM) system".

A skoro już tu jestem, to proponuję tytuł artykułu zmienić na Field Type Reference zamiast "Field Types..."

EDIT: Nevermind, znalazłem dokładnie to, o czym piszę, w kolejnym pliku.

Reply:
Re: page title, true, but this should be changed in a separate PR (all version are affected)